Base_Scripts/Scrapers/list_pdf_extractors/list_pdf_v3.py

```diff
@@ -0,0 +1,13 @@
+import sys
+import os
+import configs
+from pathlib import Path
+
+p = Path(__file__).resolve().parents[3]
+sys.path.insert(1, str(p) + "/common")
+from list_pdf_scrapers.list_pdf_v3 import list_pdf_v3
+
+cur_dir = os.getcwd()
```

Comment:
You can should be able to replace these lines with `save_dir = "./data/"`

Reply:
Updated in with fcf0ffdab